frontend/vue.config.js

```diff
@@ -1,18 +1,53 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-var-requires, no-console */
 const path = require("path");
-const { unlink } = require("fs");
+const { spawnSync } = require("child_process");
 
 const WasmPackPlugin = require("@wasm-tool/wasm-pack-plugin");
 const LicenseCheckerWebpackPlugin = require("license-checker-webpack-plugin");
 
-let rustLicenses = [];
-let debugMode = false;
-try {
-	// eslint-disable-next-line global-require, import/extensions, import/no-unresolved
-	rustLicenses = require("./rust-licenses");
-} catch (_) {
-	// Rust licenses are not generated by Cargo About except in release mode (`npm run build`)
-	debugMode = true;
+let rustLicenses = null;
+if (process.env.NODE_ENV === "production") {
+	try {
+		console.info("Generating license information for rust code");
+		const result = spawnSync("cargo", ["about", "generate", "about.hbs"], {
+			cwd: path.join(__dirname, ".."),
+			encoding: "utf8",
+			timeout: 60000, // one minute
+			shell: true,
+			windowsHide: true, // hide the DOS window on windows
+		});
+
+		if (result.status !== 0) {
+			if (result.status !== 101) {
+				// cargo returns 101 when the subcommand wasn't found
+				console.error("cargo-about failed", result.status, result.stderr);
+			}
+			throw new Error("");
+		}
+
+		// Make sure the output starts as expected, we don't want to eval an error message.
+		const CODEWORD = "GRAPHITE-LICENSES: ";
+		const stdout = result.stdout.trim();
+		if (stdout.substr(0, CODEWORD.length + 1) === `${CODEWORD}[`) {
+			// Security-wise, eval() isn't any worse than require(), but it doesn't need a temporary file.
+			// eslint-disable-next-line no-eval
+			rustLicenses = eval(stdout.substr(CODEWORD.length));
+		} else {
+			console.error("Unexpected output from cargo-about", result.stdout);
+		}
+	} catch (e) {
+		// Nothing to show. Error messages were printed above.
+	}
+
+	if (!rustLicenses) {
```

Comment:
See the comment below about changing this to `if (rustLicenses.length === 0) {`.